### ISSUE TITLE
Version check does not seem to work properly

### ISSUE THREAD

**Issue by tbekolay** [jellyfin/jellyfin-androidtv#1162]
**Describe the bug**
After my TV updated Jellyfin, I can no longer connect to my server. It gives me the following message:

> Server needs to be updated. Current version is 10.7.0 but app requires 10.7.0.0 or newer

I am using the `jellyfin/jellyfin` docker image on my server and I have pulled the latest version. This feels like an issue with parsing the version string?

**To Reproduce**
I am using the docker-compose install instructions from the official wiki's installation instructions. My docker-compose.yml is not special:

```
version: "3.5"
services:
  jellyfin:
    image: jellyfin/jellyfin
    container_name: jellyfin
    user: 1000:1000
    network_mode: "host"
    volumes:
      - /home/me/jellyfin/config:/config
      - /home/me/jellyfin/cache:/cache
      - /media/data/Media:/media/data:ro
    restart: "unless-stopped"
```

I installed the Android TV app from the app store and it automatically updated itself, I have not done anything special on my TV.

**Expected behavior**
Seems like version 10.7.0 should be >= 10.7.0.0

**Logs**
Nothing stands out to me looking through the logs, but if there's additional debug logging I can enable let me know.

**Screenshots**

![PXL_20211001_034816935](https://user-images.githubusercontent.com/203709/135562486-e220c77e-5f24-4f9c-ba03-705d99df3dd3.jpg)

**System (please complete the following information):**
 - Android TV or Fire TV version: Android 9
 - Device manufacturer: Sony
 - Device model: Bravia XBR-65X950H
 - Jellyfin server version: 10.7.0

**Comment by tbekolay** [comment]
As an update, and for anyone else experiencing this issue, there is an unstable version of the Jellyfin docker container, so doing the following got things back up and running until the issue is fixed:

```
docker pull jellyfin/jellyfin:unstable
docker restart jellyfin
```

**Comment by nielsvanvelzen** [comment]
Can you share the response you get when you open `https://[yourserver]/system/info/public` in the browser?

**Comment by tbekolay** [comment]
Unfortunately I pulled the unstable version of the docker image and I'm having trouble downgrading back to 10.7.0. I've tried removing the `latest` and `unstable` images and then pulling `latest` again but my server continues to report that it's 10.7.7 even though it used to report that it was 10.7.0 on the `latest` image. If someone knows how to downgrade precisely to version 10.7.0 it would be helpful!

Right now `https://[yourserver]/system/info/public` responds:

```
{"LocalAddress":"http://192.168.100.114:8096","ServerName":"bekolay","Version":"10.7.7","ProductName":"Jellyfin Server","OperatingSystem":"Linux","Id":"16c90135a7a44767809b9adf327c3277","StartupWizardCompleted":true}
```

I am pretty confident that the only difference in the older version was that the `"Version"` string said `"10.7.0"`, as I know that in the admin dashboard view the version was reported exactly as `10.7.0`.

Apologies for updating and making reproducing more difficult (leaving it non-functional would have incurred some wrath from my daughter). Let me know if there's anything else I can provide.

**Comment by nielsvanvelzen** [comment]
Since you've updated it won't be easy to find the cause of the issue. I think it might have been cache or something like that, as the check works for everyone else. I did however fix the toast message showing "10.7.0.0" instead of "10.7.0", although this is purely a cosmetic change.

**Comment by tbekolay** [comment]
Right, I hadn't pulled the latest docker image in quite a while, so a cache issue makes sense. I'll look into that if a similar issue reoccurs in the future. If no one else is experiencing this issue then that's great (and if they are they can reopen this and help debug).

Thanks for the quick response and all your work @nielsvanvelzen! And all the other contributors, v0.12 looks fantastic on my TV.
